src/settings/index.ts

```diff
@@ -67,6 +67,8 @@ export interface Settings {
     customQrTiles: CustomTile[]
     showCustomTiles: boolean
     hiddenCustomTileIds: string[]
+    fontSize?: number
```

Comment:
do we save font size eg 16px or font scale eg 0.5; 1; 1.5?

Reply:
i don@t understand